layouts/issues.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="mainContainer">
+    <NavBar />
+    <div class="bodyWrap">
+      <Sorter />
+      <Nuxt />
+      <div class="pages">
+        <a :class="$route.query.page <= 1 ? 'hidden' : ''" @click="changePage"
+          >Back</a
+        >
+        <h2>{{ $route.query.page }}</h2>
+        <a @click="changePage">Next</a>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import NavBar from "../components/NavBar";
+import Sorter from "../components/Sorter";
+export default {
+  components: {
+    NavBar,
+    Sorter
+  },
+  computed: {
+    currentPage: function() {
+      return this.$route.query.page;
+    }
+  },
+  methods: {
+    changePage: function(e) {
+      if (e.target.text === "Back") {
+        this.$router.push(
+          `/${this.$route.params.owner}/${this.$route.params.repo}?per_page${
```

Comment:
Found a bug here: You missed a `=` which resulted in an error when I hit "back". 

Recommendation: Probably better to keep query strings in a separate function so that you can reuse it easier without worrying for typos. Especially when you need to change the URL or syntax later, you'll be happy to have the query in a reusable function. 

```js
this.$router.push(
          `/${this.$route.params.owner}/${this.$route.params.repo}?per_page=${
            this.$route.query.per_page
          }&page=${parseInt(this.$route.query.page) - 1}`
        );
```